config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```
@@ -47,8 +48,13 @@ periodics:
       - --test-cmd-args=--testsuite=testing/experimental/storage/pod-startup/suite.yaml
       - --test-cmd-name=ClusterLoaderV2
       - --timeout=300m
-      - --use-logexporter
-      - --logexporter-gcs-path=gs://sig-scalability-logs/$(JOB_NAME)/$(BUILD_ID)
+      resources:
+        requests:
+          cpu: 2
+          memory: "2Gi"
+        limits:
+          cpu: 2
+          memory: "2Gi"
```

```suggestion
      resources:
        requests:
          cpu: 2
          memory: "4Gi"
        limits:
          cpu: 2
          memory: "4Gi"
```
Sorry I missed this. this a little lot of the test. We may to monitor the test post-migration.

---

Yes maybe "2Gi" is already too little, I'm changing it now.

---

Thanks!